src/euler/staking/StakeableEulerERC4626.sol

```diff
@@ -85,6 +85,13 @@ contract StakeableEulerERC4626 is EulerERC4626, Owned {
         stakingRewards.withdraw(amount);
     }
 
+    /// @notice Allows owner to claim all staking rewards
+    function claimReward(address recipient) public onlyOwner returns (address rewardsToken, uint256 earned) {
+        (rewardsToken, earned) = reward();
+        stakingRewards.getReward();
+        _transferRewardToken(rewardsToken, earned, recipient);
+    }
```

Comment:
Amazing that we need to do `reward() + getReward()` 🤦

Reply:
Yeah, I can't believe that they don't return the reward... Very difficult to integrate to be honest